camunda-formio-tasklist-vue/src/styles/camundaFormIOTasklist.scss

```diff
@@ -441,3 +441,17 @@ li {
   font-size: 1.4em;
 }
 }
+
+//TODO Remove this when Semantic UI css is removed. Semantic UI css override formio css tooltip
+
```

Comment:
is there any reason not to add the CSS inside ftf-form-conatiner??

Reply:
When I tried put inside scss, it's not working